pkg/plugins/golang/v3/webhook.go

```diff
@@ -77,6 +77,10 @@ func (p *createWebhookSubcommand) BindFlags(fs *pflag.FlagSet) {
 		"if set, scaffold the validating webhook")
 	fs.BoolVar(&p.options.DoConversion, "conversion", false,
 		"if set, scaffold the conversion webhook")
+	fs.BoolVar(&p.options.DoHubScaffold, "hub", false,
+		"if set, scaffold ConvertTo function for Hub conversion")
+	fs.BoolVar(&p.options.DoSpokeScaffold, "spoke", false,
+		"if set, scaffold ConvertFrom function for Spoke conversion")
```

Comment:
Could we have a hub and spoke interfaces for the other types that are not the Conversion ones?
If not, what about we add a validation?

Reply:
Also, can we have both spoke and hub as true?

Reply:
You mean if you set spoke or hub flags true, conversion flag should be true, right?
I don't think conversion flag need anymore because it just print message that says you have to implement Hub and Convertible interface [here](https://github.com/kubernetes-sigs/kubebuilder/blob/master/pkg/plugins/golang/v4/scaffolds/webhook.go#L93-L96).